From: Kalesh AP <kalesh.purayil@emulex.com>
Date: Tue, 20 Jan 2015 03:51:49 -0500
Subject: [PATCH 103/145] be2net: Fix TX rate limiting on Lancer/Skyhawk-R VFs
Patch-mainline: v4.0-rc1
Git-commit: 58bdeaa666152b2e03bb6e0d754d8186120fc103
References: bsc#921784 FATE#318561

When max_tx_rate is set via bw_max in the NIC resource desc, bw_min must be
set to 0.
Signed-off-by: Kalesh AP <kalesh.purayil@emulex.com>

Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/emulex/benet/be_cmds.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/emulex/benet/be_cmds.c
+++ b/drivers/net/ethernet/emulex/benet/be_cmds.c
@@ -3754,6 +3754,7 @@ int be_cmd_config_qos(struct be_adapter
 	be_reset_nic_desc(&nic_desc);
 	nic_desc.pf_num = adapter->pf_number;
 	nic_desc.vf_num = domain;
+	nic_desc.bw_min = 0;
 	if (lancer_chip(adapter)) {
 		nic_desc.hdr.desc_type = NIC_RESOURCE_DESC_TYPE_V0;
 		nic_desc.hdr.desc_len = RESOURCE_DESC_SIZE_V0;
